design/indexing.md

```diff
@@ -63,10 +63,10 @@ is found, the compile command for each source file is placed on a queue.
 Before indexing each file, the index checks for a cached `*.idx` file on disk.
 After indexing, it writes this file. This avoids reindexing on startup if
 nothing changed since last time.
-These files are located in `.cache/clangd/index/` next to `compile_commands.json`.
-For headers with no CDB, such as the standard library, they are in `clangd/index`
-under the user's cache directory (`$XDG_CACHE_HOME`, `DARWIN_USER_CACHE_DIR`, or
-`%LocalAppData%`).
+These files are located in `.cache/clangd/index/` next to
+`compile_commands.json`. For headers with no CDB, such as the standard
```

Comment:
FWIW, I find this kind of rewrapping makes the source code harder to read and edit, and it has no effect on the output. (Analogy: paragraphs are like functions, line breaks in the source are like blank lines in functions, separating thoughts)
But up to you.